airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/LabelInfo.tsx

```diff
@@ -20,12 +21,39 @@ const Description: React.FC<Pick<LabelInfoProps, "label" | "description">> = ({
   return (
     <div>
       {/* don't use <Text as=h4> here, because we want the default parent styling for this header */}
-      <h4 className={styles.descriptionHeader}>{label}</h4>
+      <h3 className={styles.descriptionHeader}>{label}</h3>
       <TextWithHTML className={styles.description} text={description} />
     </div>
   );
 };
 
+const Options: React.FC<Pick<LabelInfoProps, "options">> = ({ options }) => {
+  if (!options) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h4 className={styles.optionsHeader}>
+        <FormattedMessage id="connector.optionsHeader" />
+      </h4>
+      <ul className={styles.options}>
+        {options.map((option) => (
```

Comment:
Do you think it would make sense to only render this options list if one of the options has a `description`?

I'm suggesting this due to cases like this:
<img width="716" alt="Screen Shot 2022-12-02 at 10 37 37 AM" src="https://user-images.githubusercontent.com/22731524/205363178-310d2221-aaab-408a-bb19-22d82c1c0c0c.png">
where none of the options have any descriptions, in which the `Options` list in the tooltip doesn't really provide any more info than clicking on the dropdown to see which options are available, so it feels somewhat redundant.

Reply:
You're reasoning makes sense to me, I'm on board with keeping it! Thanks